youtube_dl/extractor/w24at.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+from __future__ import unicode_literals
+import re
+
+from .common import InfoExtractor
+
+
+class W24atIE(InfoExtractor):
+    _VALID_URL = r'https://(?:www\.)?w24\.at/Video/.*/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://www.w24.at/Video/Bewegung-macht-Spass-Folge-62-Kids-6/24828',
+        'md5': '16e6f1c5d4a0d54d420e3d9d122660a1',
+        'info_dict': {
+            'id': '24828',
+            'ext': 'mp4',
+            'title': 'Bewegung macht Spaß! - Folge 62: Kids 6',
+            'description': 'Stefans Ziel ist es Kindern auch hinter den Bildschirmen zur Bewegung und zum Denksport zu animieren und das ganze mit Spaß und Köpfchen zu verbinden.',
+            'thumbnail': r're:.*\.jpg$',
+            'uploader': 'W24'
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        media_server = self._html_search_regex(r'var\s*mediaServer\s*=\s*\{.*"vod"\s*:\s*"([^"]+)"',
+                                               webpage, "vod", "ms02.w24.at")
+        mp4_path = self._html_search_regex(r"src:.*\+ '([^']+)'.*type:'video/mp4'",
+                                           webpage, "mp4_path")
+        m3u8_path = self._html_search_regex(r"src:.*\+ '([^']+)'.*type:'application/x-mpegURL'",
+                                            webpage, "m3u8_path")
+        formats = []
+        if mp4_path:
+            formats.append({'url': "https://%s%s" % (media_server, mp4_path), 'ext': 'mp4'})
+        formats.extend(self._extract_m3u8_formats("https://%s%s" % (media_server, m3u8_path), video_id, ext='mp4'))
```

Comment:
I'm not saying this is much better but it might be a bit more resilient to format changes:
```suggestion
        media_server = self._parse_json(
            self._search_regex(r'mediaServer\s*=\s*(\{[^}]+})\s*;', webpage, 'media servers', default='{}'),
            video_id)
        src_txt = self._search_regex(r'videoPlayer%s\.src\s*\(\s*(\[[^\]+])' % (video_id, ), webpage, 'media paths')
        srcs = []
        try:
            srcs = eval(
                src_txt.replace('mediaServer.vod', 'vod').replace(',\n',', '),
                {'vod': media_server.get('vod', 'ms02.w24.at'), 'src': 'src', 'type': 'type'})
            # expect a non-empty list 
            srcs[0]    
        except Exception as e:
            raise ExtractorError('Unable to extract media links', cause=e)
        formats = []
        for src in srcs:
            fmt_url = url_or_none(src.get('src'))
            if not fmt_url:
                continue
            ext = mimetype2ext(src.get('type')) or determine_ext(fmt_url)
            if ext == 'm3u8':
                formats.extend(self._extract_m3u8_formats(
                    fmt_url, video_id, 'mp4',
                    entry_protocol='m3u8_native', m3u8_id='hls', fatal=False))
            else:
                formats.append({
                    'url': fmt_url,
                    'ext': ext,
                })
```